.gitlab-ci.yml

```diff
@@ -0,0 +1,90 @@
+image: golang:1.11
```

Comment:
Should we use a newer golang for this service? As it is new, I would suggest trying with latest golang 1.13